Toggl.Droid/Fragments/SettingsFragment.cs

```diff
@@ -164,6 +167,20 @@ public override void OnViewCreated(View view, Bundle savedInstanceState)
             smartRemindersView.Rx().Tap()
                 .Subscribe(ViewModel.OpenCalendarSmartReminders.Inputs)
                 .DisposedBy(DisposeBag);
+
+            ViewModel.CurrentSyncStatus
+                .Subscribe(setSyncStatusView)
+                .DisposedBy(DisposeBag);
+        }
+
+        private void setSyncStatusView(PresentableSyncStatus status)
+        {
+            syncStateViews.Values.ForEach(view => view.Visibility = ViewStates.Gone);
+
+            txtStateInProgress.Text = status == PresentableSyncStatus.Syncing ? Syncing : LoggingOutSecurely;
+
+            syncStateViews.TryGetValue(status, out var visibleView);
```

Comment:
If you are not using the `bool` return value to check for existance of the key, please use the dictionary access syntax.
```suggestion
            var visibleView = syncStateViews[status];
```